### PR TITLE
setting up pr-labeler action

### PR DESCRIPTION
**Pull request by mschwrdtnr**
Setting up an action to automatically label pull requests based on branch name

feel free to add something

Using: https://github.com/marketplace/actions/pr-labeler

### CODE REVIEW

File: .github/pr-labeler.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+feature: ['feature/*', 'feat/*']
+restyled: ['restyled/*]
+demo: ['demo/*']
```
Comment (DanielHabenicht) [inline .github/pr-labeler.yml:3]: have we ever used that? ^^

Can one for `test` and `chore`?

Comment (mschwrdtnr) [reply .github/pr-labeler.yml:3]: Do you mean "demo"? I am not sure, but why not.. my only motivation was it to label the automatically created restyled PRs, so i thought I also add the other things^^

The requirement that this action bring value is that everyone is creating meaningful branches..

Comment (mschwrdtnr) [reply .github/pr-labeler.yml:3]: We haven't a label for "test". I will add the code, if you add the label^^

Comment (DanielHabenicht) [reply .github/pr-labeler.yml:3]: Just fyi restyled could label it's branches it self :D

**Comment by mschwrdtnr** [comment]
@friedaxvictoria you can also take a look if you want :)

**Pull request merged**